osc-server/src/main/java/org/osc/core/broker/model/sdn/PortPairGroupNetworkElementImpl.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.model.sdn;
+
+import java.util.List;
+
+import org.osc.sdk.controller.element.NetworkElement;
+
+public class PortPairGroupNetworkElementImpl implements NetworkElement {
```

Comment:
can you add java doc to indicate this just contains element id and uses that to compare?

In general for PortPairGroups the parent id will reflect the sfc id...since this is at odds with this class doing the above makes it clear

Reply:
Done